src/StaticFAQ/FAQArticleDetail.js

```diff
@@ -11,10 +11,22 @@ import { LanguageContext } from '../context/Language';
 import type { FAQArticleDetailQuery } from './__generated__/FAQArticleDetailQuery.graphql';
 
 const queryFAQArticleDetail = graphql`
-  query FAQArticleDetailQuery($id: ID!, $language: Language) {
+  query FAQArticleDetailQuery(
+    $id: ID!
+    $language: Language
+    $category_id: ID!
+  ) {
     FAQArticle(id: $id, language: $language) {
       ...FAQArticleDetailContent_article
     }
+    FAQCategory(id: $category_id) {
+      id
```

Comment:
it fetches `id`, `title` and `ancestors.id` for this component but it's never used